app/main.py

```diff
@@ -1 +1,48 @@
-# write your code here
+import random
+import math
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+
+
+def flip_coin():
+    array_of_heads = [
+        sum([random.randint(0, 1) for _ in range(10)])
+        for _ in range(10000)
+    ]
+
+    result_dict = {}
+    for x in range(11):
+        result_dict[x] = math.ceil((array_of_heads.count(x) / 10000) * 100)
```

Comment:
maybe rename x in something like head_count

Reply:
agree